Modules/Microsoft365DSC/Modules/M365DSCPermissions.psm1

```diff
@@ -1205,6 +1218,57 @@ If you want to configure App-Only permission for Exchange, as described here:
 Using the following permission will achieve exactly that: @{Api='Exchange';PermissionsName='Exchange.ManageAsApp'}
 
+Note 2:
+If you want to configure App-Only permission for Security and compliance, please refer to this information on how to setup the permissions:
+https://microsoft365dsc.com/user-guide/get-started/authentication-and-permissions/#security-and-compliance-center-permissions
+
+Note 3:
+If you want to configure App-Only permission for Power Platform, please refer to this information on how to setup the permissions:
+https://microsoft365dsc.com/user-guide/get-started/authentication-and-permissions/#power-apps-permissions
+
+
+.Parameter ApplicationName
+The name of the application to create or update. Default value is 'Microsoft365DSC'.
+
+.Parameter Permissions
+The permissions to assign to the application. This has to be an array of hashtables, with Api=Graph, SharePoint or Exchange and PermissionsName set to a list of permissions. See examples for more information.
+
+.Parameter Type
+The type of credential to create. Default value is 'Secret'. Valid values are 'Secret' and 'Certificate'.
+
+.Parameter MonthsValid
+The number of months the certificate should be valid. Default value is 12.
+
+.Parameter CreateNewSecret
+If specified, a new secret will be created for the application. -CreateNewSecret or -CertificatePath can be used, not both.
+
+.Parameter CertificatePath
+The path to the certificate to use for the application. -CreateNewSecret or -CertificatePath can be used, not both.
+
+.Parameter CreateSelfSignedCertificate
+If specified, a self-signed certificate will be created for the application. -CreateSelfSignedCertificate or -CertificatePath can be used, not both.
+
+.Parameter AdminConsent
+If specified, admin consent will be granted for the application.
+
+.Parameter Credential
+The credential to use for authenticating the request.
+
+.Parameter ApplicationId
+The ApplicationId to use for authenticating the request. -Credential or -ApplicationId can be used, not both.
+
+.Parameter TenantId
+The TenantId to use for the request. The TenantId needs to be in the form of contoso.onmicrosoft.com. -Credential or -TenantId can be used, not both.
+
+.Parameter ApplicationSecret
+The ApplicationSecret to use for authenticating the request. -Credential or -ApplicationSecret can be used, not both.
+
+.Parameter CertificateThumbprint
+The CertificateThumbprint to use for authenticating the request. -Credential or -CertificateThumbprint can be used, not both.
+
+.Parameter ManagedIdentity
+If specified, Managed Identity will be used for authenticating the request. -Credential or -ApplicationId or -ManagedIdentity can be used, only one of them.
+
 .Example
 Update-M365DSCAzureAdApplication -ApplicationName 'Microsoft365DSC' -Permissions @(@{Api='SharePoint';PermissionName='Sites.FullControl.All'}) -AdminConsent -Type Secret -Credential $creds
```

Comment:
Maybe, break up the examples into multiple lines for better readability?
```suggestion
Update-M365DSCAzureAdApplication -ApplicationName 'Microsoft365DSC' `
-Permissions @(@{Api='SharePoint';PermissionName='Sites.FullControl.All'}) `
-AdminConsent `
-Type Secret `
-Credential $creds
```

Reply:
Line breaks are not working with our tooling to auto generate the documentation